doc/api/deprecations.md

```diff
@@ -914,6 +914,16 @@ Type: Runtime
 
 This was never a documented feature.
 
+<a id="DEP01XX"></a>
+### DEP01XX: process.env string coercion
+
+Type: Documentation-only (supports [`--pending-deprecation`][])
```

Comment:
I'd move the supports `--pending-deprecation` to the description below.

Reply:
@jasnell Just fyi, this was pretty recently added to the deprecation type on purpose in https://github.com/nodejs/node/pull/18433